set.mm

```diff
@@ -33784,6 +33784,18 @@ general is seen either by substitution (when the variable ` V ` has no
       ( wcel cv wceq wa wrex r19.42v eleq1 adantr pm5.32ri bicomi baib ceqsrexv
       wb rexbiia pm5.32i 3bitr3i ) DEGZCHZDIZAJZJZCEKUCUFCEKZJUHUCBJUCUFCELUGUF
       CEUGUDEGZUFUIUFJUGUFUIUCUEUIUCSAUDDEMNOPQTUCUHBABCDEFRUAUB $.
+
+    $( Alternate elimitation of a restricted existential quantifier, using
+       implicit substitution.  (Contributed by Scott Fenton, 5-Sep-2017.) $)
+    ceqsrexv2 $p |- ( E. x e. B ( x = A /\ ph ) <-> ( A e. B /\ ps ) ) $=
+      ( ceqsrexbv ) ABCDEFG $.
```

Comment:
ceqsrexv2 and ceqsrexbv are duplicates

Reply:
Yes, the 'b' for 'biconditional' makes indeed for better labels for these two theorems.

Reply:
@icecream17 oops good catch, thanks. It was right under there and I didn't notice! Deleted. @icecream17 @benjub: *bv: thank you, fixed.